src/sysl/importers/import_swagger.py

```diff
@@ -330,18 +399,48 @@ def r(t):
         elif (typ, fmt) == ('number', None):
             return r('float')
         elif (typ, fmt) == ('object', None):
-            descrs = {
-                k: descr
-                for (k, v) in extract_properties(tspec).iteritems()
-                for descr in [self.parse_typespec(v)[1]]
-                if descr
-            }
-            assert not descrs, descrs
-            fields = ('{} <: {}'.format(k, self.parse_typespec(v)[0])
-                      for (k, v) in sorted(extract_properties(tspec).iteritems()))
-            return r('{' + ', '.join(fields) + '}')
+            typeSpecList.append(TypeSpec(tspec, parentRef, typeRef))
+            retVal = typeRef + ('_' + parentRef if len(parentRef) > 0 else '') + '_obj'
+            if ('properties' not in tspec):
+                return r('EXTERNAL_' + retVal)
+            return r(retVal)
+
         else:
-            return r(str(tspec))
+            aliasName = 'EXTERNAL_' + typeRef + ('_' + parentRef if len(parentRef) > 0 else '') + '_obj'
+            externAlias[aliasName] = 'string'
+            return r(aliasName)
+
+    def getTag(self, tagName, tagType):
+        if tagType is None or tagName is None:
+            return ''
+        if tagType == 'json':
+            return '\n\t@json_tag = "{}"'.format(tagName)
+        return '\n\t@{} = "{}"'.format(tagType, tagName)
+
+    def getHeaders(self, headerParams):
+        headerList = []
+        alias = {}
+
+        for headerParam in headerParams:
+            paramName = headerParam['name']
+            typeName = paramName.replace('-', '_')
+            if 'required' in headerParam and headerParam['required']:
+                headerList.append('{} <: {} [~header, ~required, name="{}"]'.format(typeName.lower(), typeName, paramName))
+            else:
+                headerList.append('{} <: {} [~header, ~optional, name="{}"]'.format(typeName.lower(), typeName, paramName))
+            alias[typeName] = headerParam['type']
```

Comment:
I don't think we need to generate `!alias` for headers. 
Type should be whatever is specified in swagger. If not specified, make it string.